tools/apiClient.py

```
@@ -0,0 +1,147 @@
+    path = argv[2]
+    if not os.path.exists(path):
+        print("no such file: %s" % path)
+        sys.exit(2)
+    fields = {}
+    if verb == "analyze" and len(argv) >= 4:
+        fields["m"] = str(int(argv[3]))
+    body, ctype = multipart(fields, "file", path)
+    req = urllib.request.Request(base + "/" + verb, data=body, method="POST",
+                                 headers={"Authorization": "Bearer " + token,
+                                          "Content-Type": ctype,
+                                          "Content-Length": str(len(body))})
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)